addons/interaction/functions/fnc_getWeaponAttachmentsActions.sqf

```diff
@@ -113,7 +113,7 @@ params ["_unit"];
                                 LINKFUNC(switchWeaponAttachment),
```

Comment:
Wouldn't it make more sense to replace the function with `EFUNC(common,switchAttachmentMode)` instead of adding a parameter to `ace_interaction_fnc_switchWeaponAttachment`?